README.md

```diff
@@ -21,9 +21,9 @@ API Repository to describe, develop, document, and test the NumberRecycling API
 
 * Note: Please be aware that the project will have updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. For best results, use the latest available release.
 
-* **The latest pre-release of CAMARA Number Recycling is [r1.1](https://github.com/camaraproject/NumberRecycling/releases/tag/r1.1)**. The Release Tag is [r1.1](https://github.com/camaraproject/NumberRecycling/releases/tag/r1.1).
+* **The latest public release of CAMARA Number Recycling is [r1.2](https://github.com/camaraproject/NumberRecycling/releases/tag/r1.2)**. The Release Tag is [r1.2](https://github.com/camaraproject/NumberRecycling/releases/tag/r1.2).
```

Comment:
The line is correct, just as a suggestion maybe to emphasize that it is the first public release:

```suggestion
* **The first public release of CAMARA Number Recycling is [r1.2](https://github.com/camaraproject/NumberRecycling/releases/tag/r1.2)**. The Release Tag is [r1.2](https://github.com/camaraproject/NumberRecycling/releases/tag/r1.2).
```